src/components/30-organisms/commercial-hero-banner/README.md

```diff
@@ -117,10 +117,126 @@ Import the commercial-hero-banner-defining script and use a commercial-hero-bann
   <h1 slot="title">...</h1> // Only one title
   <p slot="content">...</p> // You can put multiple <p> elements with the 'content' slot
   <small slot="disclaimer">...</small> // Only one disclaimer
-  <axa-button-link slot="button" /> // You can put multiple <p> elements with the 'content' slot
+  <axa-button-link slot="button" /> // Can also be 'axa-button', meant as CTA (Call to Action)
+  <div slot="badge">...</div> // Any DOM node, meant to visualize a badge displayed at the upper right position
```

Comment:
Please use a more flexible name like: `top-right-section` and `bottom-right-section`